Feature/kemenov/dp-2-setup-cicd PR

SKemenov opened the pull request:
- Влил файлы конфигурации fastlane в папку `./fastlane`
- Влил файлы конфигурации GitHub Actions в папку `./.github/workflows`
- Запуск fastlane локально отрабатывает, сборка собирается без ошибок:

[21:08:19]: ▸ Test build Succeeded

| Step | Action | Time (in s) |
| --- | --- | --- |
| 1    | default_platform | 0           |
| 2    | scan             | 9           |

[21:08:19]: fastlane.tools finished successfully 🎉

SKemenov committed:
Установил локально fastlane, настроил GitHub Actions yaml-файлы

superaydumov commented on .github/workflows/gha-pr-dev.yml:
```diff
@@ -0,0 +1,29 @@
+  pull_request:
+    branches:
+      - 'feature/**'
+      - 'bugfixe/**'
```
Мне кажется, что тут опечатка. Проверь пожалуйста

SKemenov replied:
Исправил

SKemenov committed:
Внес правки по ревью

- Исправил название ветки на `bugfix/`.
- Убрал параметр '-strict` в запуске Swiftlint.

IuriiKarpunov reviewed (approved):
Опечаток нет, файлы выглядят хорошо структурированными
